### PR TITLE
Lesson 4 2

### PR DESCRIPTION
**Pull request by Raven1223**

### CODE REVIEW

**Comment by jacobboney** [comment]
I think some of the issues you're having have to do with the formatting of your tags. I would recommend revisiting the previous assignment (4.1) and going over where you placed your tags. 

For example:
- You're missing the closing header tag
- You're missing a few closing section tags
- The location of you're script tag shouldn't be inside of the footer tag
- Missing the closing tag for <ul> (I think this should solve your problems)

I think if you go through and format your tags, you should be all set. I looked over your index.js file and everything looks good. I'll also add that I made these changes that I was speaking about before and everything seems to be working fine. I also noticed a small typo in your for loop when you're selecting a skill as well as in your query selector when referencing skillsSection.

Basically, go back over your code with a fine tooth comb and I think you'll do just fine. You executed the functionality greatly, there are just some small typos and formatting to take care of.

Great Work!
